clients/python/girder_client/__init__.py

```diff
@@ -168,8 +168,29 @@ class GirderClient(object):
 
     DEFAULT_API_ROOT = 'api/v1'
     DEFAULT_HOST = 'localhost'
-    DEFAULT_PORT = 443
-    DEFAULT_SCHEME = 'https'
+    DEFAULT_HTTP_PORT = 8080
```

Comment:
After installing Girder, 8080 is the default (always have been from what I can tell):
* https://github.com/girder/girder/blob/aa0ff5092cd570958fd379ddf8149c280521df5f/docs/installation.rst#run
* https://github.com/girder/girder/blob/aa0ff5092cd570958fd379ddf8149c280521df5f/girder/conf/girder.dist.cfg#L3